docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,83 @@
-
-First, we pause all incoming requests and hold them so that there's no interruption of service.
-Then we disconnect the current containers from their file system mounts, if any,
-and connect the file systems to the new containers instead.
-If it's a new branch and there is no existing file system, we clone it from the parent branch.
-
-We then open networking connections between the various containers,
-but only those that were specified in the configuration files (using the `relationships` key).
-No configuration, no connection.
-That helps with security, as only those connections that are actually needed even exist.
-The connection information for each service is available in an application as environment variables.
-
-Now that we have working, running containers there are two more steps to run.
-First, if there is a `start` command specified in your `.platform.app.yaml` file to start the application, we run that.
-(With PHP, this is optional.)
-
-Then we run your deploy hook.
-Just like the build hook, the deploy hook is any number of shell commands you need to prepare your application.
-Usually this includes clearing caches, running database migrations, and so on.
-You have complete access to all services here as your application is up and running,
-but the file system where your code lives is now read-only.
-
-Finally, once the deploy hooks are done,
-we open the floodgates and let incoming requests through to your newly deployed application.
-You're done!
-
-In practice, the deploy process takes only a few seconds, plus whatever time is required for your deploy hook, if any.
+Each time you push a change to your app through Git or activate an [environment](../administration/web/environments.md),
+your app goes through a process to be built and deployed.
+If your app is redeployed with no changes, the output of the previous build and deploy process is used.
+
+The build process looks through the configuration files in your repository and assembles the necessary containers.
+The deploy process makes those containers live, replacing any previous versions, with minimal interruption in service.
+
+![The steps in the build and deploy process](/images/workflow/build-pipeline.svg "0.50")
+
+## Glossary
+
+Hooks
+: Hooks are points in the build and deploy process where you can inject a custom script.
+
+## The build
+
+The outcome of the build process is designed to be repeatable and reusable.
+Each app in a project is built separately.
+
+Container configuration depends exclusively on your configuration files.
+So each container is tied to a specific Git commit.
+If there are no new changes for a given container, the existing container can be reused.
+This saves you the time of the build step.
+
+This means the build is independent of the given environment and development environments are perfect copies of production.
+
+This independence does mean you can't connect to services (like databases) during the build.
```

Comment:
```suggestion
You can connect to services (like databases) during the build step.
```
I'm trying to simplify what someone can do, and dropped the reference to the build independence from above. I think this might work better with the line below also. Would you agree?

Reply:
I like the simplification. I'd commit the change, but it says you can connect during the build step, which I don't think is true.

Reply:
Thanks for addressing the simplification and making it truthful 😅  not sure how I managed to change its meaning so much!